lib/routes/app-sales/util.ts

```diff
@@ -0,0 +1,160 @@
+import { type DataItem } from '@/types';
+
+import { art } from '@/utils/render';
+import ofetch from '@/utils/ofetch';
+
+import { type CheerioAPI, type Cheerio, type Element, load } from 'cheerio';
```

Comment:
```suggestion
import { type CheerioAPI, type Cheerio, load } from 'cheerio';
import type { Element } from 'domhandler';
```
cheerio no longer re-export Element [since v1.0.0](https://github.com/cheeriojs/cheerio/releases/tag/v1.0.0).

Reply:
Thanks for the review. 😊